converter.cpp

```diff
@@ -260,7 +325,7 @@ void convert(const std::string& wav_file_str, const std::string& s_file_str,
     if (ct == cmp_type::none)
         convert_uncompressed(wf, fout);
     else if (ct == cmp_type::dpcm)
-        convert_dpcm(wf, fout);
+        convert_dpcm(wf, fout, verbose);
```

Comment:
I suggest to replace verbose parameter with a global variable like the other program parameters (I prefer this for readability if the values don't change during operation).